service_msgs/README.md

```diff
@@ -0,0 +1,13 @@
+# service_msgs
+This package contains message types useful for ros2 services.
+
+More information about services can be found on its [design article](http://design.ros2.org/articles/actions.html).
```

Comment:
This tutorial might be the best place to reference for an intro to ROS services: https://docs.ros.org/en/foxy/Tutorials/Beginner-CLI-Tools/Understanding-ROS2-Services/Understanding-ROS2-Services.html

Reply:
[f274555](https://github.com/ros2/rcl_interfaces/pull/143/commits/f27455529315fbbc6f49a16f852156e379bab2c5)